.gitlab-ci.yml

```diff
@@ -0,0 +1,106 @@
+include:
+  - remote: 'https://gitlab-templates.ddbuild.io/dd-package/v1/template.yml'
+  - remote: 'https://gitlab-templates.ddbuild.io/slack-notifier/v1/template.yml'
+
+stages:
+  - ci-image
+  - assemble
+  - build
+  - release
+  - notify
+
+.gradle-job:
+  image: openjdk:8
+  tags: ['runner:main', 'size:large']
+  before_script:
+    - export GRADLE_USER_HOME=`pwd`/.gradle
+    - echo $GRADLE_USER_HOME
+  cache:
+    paths:
+      - .gradle/wrapper
+      - .gradle/caches
+  variables:
+    GRADLE: "./gradlew -I gradle/init-publish.gradle --no-daemon --max-workers=1"
+
+# I don't think this is necessary, if anything we'd maybe run assemble or something
+# but its going to change on nearly any file change, so we lose our caching
+# maybe using it as a precooked gradle cache? but at that point why not jsut use gitlabs
+# caches?
+ci-image:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:18.03.1
+  stage: ci-image
+  only:
+    changes:
+      - Dockerfile
+    refs:
+      - datadog-build
+      - engtools/243/gitlab_builds_for_clouddriver
+  tags: ['runner:docker', 'size:large']
+  script:
+    - docker build --tag 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/clouddriver:latest .
+    - docker push 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/clouddriver:latest
+
+assemble:
+  extends: .gradle-job
+  stage: assemble
+  script:
+    - $GRADLE assemble
+
+build:
+  extends: .gradle-job
+  stage: build
+  script:
+    - $GRADLE build
+
+# TODO set version off of tag -Prelease.useLastTag
+build-deb:
+  extends: .gradle-job
+  stage: build
+  artifacts:
+    paths:
+      - ./clouddriver-web/build/distributions/*.deb
+  script:
+    - $GRADLE buildDeb -x test
+
+build-all:
+  extends: .gradle-job
+  stage: build
+  artifacts:
+    paths:
+      - ./clouddriver-web/build/distributions/*.deb
+  script:
+    - $GRADLE buildDeb build -Prelease.useLastTag
+
+release-deb:
+  extends: .dd-package.upload-debian-package
+  # XXX remove before merge
+  only:
+    - engtools/243/gitlab_builds_for_clouddriver
+  # only run this on tags on commits that are on the datadog-build branch
+  # XXX UNCOMMENT BEFORE MERGE
+  # only:
+  #   - tags
+  #  except:
+  #  - /^(?!datadog-build).+@/
+  dependencies:
+    - build-deb
+  variables:
+    DEB_PATH: './clouddriver-web/build/distributions/*.deb'
+
+failure:
```

Comment:
Was this stage not included by the slack notifier template?

Reply:
uh, i am either bad at git or was juggling too many repos last night, this should've been replaced by the slack-notifier component, will check that, fix it and push a fix